components/angular-tomitribe-tags/src/tags.module.ts

```diff
@@ -5,10 +5,12 @@ import {TagsDirective} from './tags.directive';
 import {TagsOnBlurDirective} from './tags-onblur.directive';
 
 import * as angular from 'angular';
+import {TagsValidatorDirective} from "./tags.validator.directive";
 
 angular.module('tomitribe-tags', ['ui.select', 'ngSanitize'])
   .service('TribeTagsService', TagsService)
   .service('TribeTagsConfigurer', TagsConfigurer)
   .controller('TribeTagsController', TagsController)
   .directive('tribeTags', [function () { return new TagsDirective(); }])
-  .directive('tribeTagsOnBlur', ['$timeout', t => new TagsOnBlurDirective(t)]);
+  .directive('tribeTagsOnBlur', ['$timeout', t => new TagsOnBlurDirective(t)])
+  .directive('tribeTagsValidator', [()=> { return new TagsValidatorDirective(); }]);
```

Comment:
I discussed this part with Joao and I couldn't get the idea why we need this directive. He described that it will e reused through other tag-like items so I asked him to rename it a bit so if we use it with groups we will understand why it works as such.